wp-content/themes/reconasia/assets/_scss/components/_post-meta.scss

```diff
@@ -29,17 +29,22 @@
   }
 
   &__date:not(:only-child) {
-    margin-left: rem(8);
+    margin-top: rem(4);
+    color: $color-black-165;
+    @extend %text-style-label-small-strong;
+  }
 
-    &::before {
-      content: '|';
-      margin-right: rem(8);
-      color: var(--post-meta-divider);
-    }
+  &__categories {
```

Comment:
Let's ask Christina in the Recon Asia slack channel if this should have a hover state.